packages/jest-transform/src/types.ts

```diff
@@ -52,21 +52,70 @@ export interface CacheKeyOptions extends TransformOptions {
   rootDir: string;
 }
 
-export interface Transformer {
+interface SyncTransFormer {
   canInstrument?: boolean;
-  createTransformer?: (options?: any) => Transformer;
+  createTransformer?: (options?: any) => SyncTransFormer;
 
   getCacheKey?: (
-    fileData: string,
+    fileDate: string,
```

Comment:
No, it's `data` 😃 could call it `content` I guess?

Reply:
Oh, my bad for the typo! 
Yes, in `ScriptTransformer`, it's sometimes referred to as `content`, seems to be a clearer name :)